woocommerce/woocommerce-ios
# Update Zendesk to version 3.0.2

Bump Zendesk to 3.0.2

## Testing
- Checkout the branch, build the app
- Send a message via Zendesk, reply to it and check that the reply is received. In other words, check that Zendesk is still working 🤷‍♂ 

Update release notes:
